apps/src/applab/DesignToggleRow.jsx

```diff
@@ -49,11 +58,16 @@ module.exports = React.createClass({
 
     if (this.state.mode === Mode.DESIGN) {
       var options = this.props.screens.map(function (item) {
-        return <option>{item}</option>;
+        return <option key={item}>{item}</option>;
       });
 
       selectDropdown = (
-        <select id="screenSelector" style={dropdownStyle}>
+        <select
+          id="screenSelector"
+          style={dropdownStyle}
+          value={this.state.activeScreen}
+          onChange={this.handleScreenChange}
+          disabled={Applab.isRunning()}>
```

Comment:
Hmm. That means we would need to make sure we rerender every time isRunning changes, but I think that should already be happening already, so you may be right. I'll confirm that this works properly.